src/documents/ModelRelay.md

```diff
@@ -134,20 +145,48 @@ This is done using the `modelRelay` section of a component's top-level options.
-            component participates in, and <code>liveOnly</code> will only have it operated after construction of the component (at all times other than the initial transaction).</td>
-            <td><code>initOnly</code></td> 
+            <td><code><a href="#-sourcefilterrecord-in-an-explicit-model-relay-block">sourceFilterRecord</a></code></td>
+            <td>Control the situations in which the <a href="#controlling-propagation-through-a-relay-rule">forward leg</a> of the transform operates - by listing one or more <a href="ChangeApplierAPI.md#source-tracking-and-filtering-in-model-listener-blocks">change sources</a> which must match (<code>includeSource</code>) or not match (<code>excludeSource</code>) in the current transaction.</td>
+            <td><code>{excludeSource: "init"}</code></td> 
         </tr>
         <tr>
             <td><code>backward</code> (optional)</td>
-            <td>One of the strings <code>never</code>, <code>initOnly</code>, <code>liveOnly</code> or <code>always</code> (the default)</td>
-            <td>Control the situations in which the <a href="#controlling-propagation-through-a-relay-rule">backward leg</a> of the transform operates &#8212; assuming that it is already an invertible transformation. The same meaning for the codes as for <code>forward</code>
-            <td><code>liveOnly</code></td>
+            <td><code><a href="#-sourcefilterrecord-in-an-explicit-model-relay-block">sourceFilterRecord</a></code></td>
+            <td>Control the situations in which the <a href="#controlling-propagation-through-a-relay-rule">backward leg</a> of the transform operates - by listing one or more <a href="ChangeApplierAPI.md#source-tracking-and-filtering-in-model-listener-blocks">change sources</a> which must match (<code>includeSource</code>) or not match (<code>excludeSource</code>) in the current transaction.</td>
+            <td><code>{excludeSource: "init"}</code></td>
+        </tr>
+        <tr>
+            <td><code>namespace</code> (optional)</td>
+            <td><code>String</code></td>
+            <td>Identifies this model relay rule uniquely within its parent component. This namespace may be used to override the relay rule from a derived <a href="ComponentGrades.md">grade</a>, or to control the priority of
+            one relay rule with respect to another by using the <code>priority</code> field. 
+            Compare with the similar uses of namespaces within Infusion <a href="InfusionEventSystem.md#full-listener-record-form">events</a>, <a href="InfusionEventSystem.md#full-listener-record-form">model listeners</a>, etc. As with `modelListener`, this
+            value will be taken from the key of the <code>modelRelayBlock</code> if it is in the long form.</td>
```

Comment:
Can you elaborate this line: `As with modelListener, this value will be taken from the key of the modelRelayBlock if it is in the long form.`?

BTW, replace the markdown syntax around **`modelListener`** with `<code>` tag.